test/fixtures/prx_story_all.json

```diff
@@ -101,6 +172,10 @@
               "profile": {
                 "href": "http://meta.prx.org/model/audio-file"
               },
+              "prx:storage": {
+                "href": "s3://mediajoint.production.prx.org/public/audio_files/1200648/lcs_spring16_act1.mp3",
+                "type": "audio/mpeg"
```

Comment:
I'm remembering that evaporate uploads (at least from Chrome) will`audio/mp3` in the temp S3 bucket.  And the cms-audio-lambda will either copy that, or detect `ffprobe.format = mp3` ... and set it to `audio/mp3` anyways.

Is that incorrect?  Should we be using the probably-more-compatible `audio/mpeg` there too?